src/main/java/ch/njol/skript/variables/VariablesStorage.java

```diff
@@ -97,11 +105,11 @@ public abstract class VariablesStorage implements Closeable {
 	 * This will also create the {@link #writeThread}, but it must be started
 	 * with {@link #load(SectionNode)}.
 	 *
-	 * @param name the name.
+	 * @param type the name.
```

Comment:
if you're going to rename the param, the javadocs should be updated (goes for all instances)